internal/scraper/nigella.go

```diff
@@ -0,0 +1,42 @@
+package scraper
+
+import (
+	"github.com/PuerkitoBio/goquery"
+	"github.com/reaper47/recipya/internal/models"
+	"strings"
+)
+
+func scrapeNigella(root *goquery.Document) (models.RecipeSchema, error) {
+	rs := models.NewRecipeSchema()
+
+	name, _ := root.Find("meta[property='og:title']").Attr("content")
+	before, _, ok := strings.Cut(name, " — ")
+	if ok {
+		name = strings.TrimSpace(before)
+	}
+	rs.Name = name
+
+	rs.Yield.Value = findYield(root.Find("p[class='serves']").Text())
+
+	untrimmedDescription, _ := root.Find("meta[property='og:description']").Attr("content")
+	description, _ := strings.CutSuffix(untrimmedDescription, "\n\nFor US cup measures, use the toggle at the top of the ingredients list.")
+	rs.Description.Value = description
+	rs.Image.Value, _ = root.Find("meta[property='og:image']").Attr("content")
+
+	keywords, _ := root.Find("meta[itemprop='keywords]").Attr("content")
+	rs.Keywords.Values = keywords
```

Comment:
One final thing: the test should be updated with the expected keywords, i.e.
```go
Keywords:    &models.Keywords{Values: "Vegetarian, Gluten Free, Nibbly, Impatient, Express, Summer, Dairy Free, Egg Free, Nut Free, Vegan"},
```

Then, you can populate the field:
```go
keywords, _ := root.Find("meta[itemprop='keywords']").Attr("content")
rs.Keywords.Values = strings.TrimSuffix(keywords, ",")
```

The reason why the test currently passes is because the missing single quote in `"meta[itemprop='keywords]"` prevents the element from being found.

Reply:
Good catch, fixed